src/platforms/python/guides/sanic/index.mdx

```diff
@@ -7,14 +7,13 @@ description: "Learn about using Sentry with Sanic."
 
 _(New in version 0.3.6)_
 
-The Sanic integration adds support for the [Sanic Web Framework](https://github.com/huge-success/sanic). We support the following versions:
+The Sanic integration adds support for the [Sanic Web Framework](https://sanicframework.org). We support the following versions:
 
 - `0.8`
 - `18.12`
 - `19.12`
 - Any version of the form `x.12` (LTS versions).
-
-**We do not support the latest version of Sanic.** Versions between LTS releases [have introduced breaking changes in the past](https://github.com/huge-success/sanic/issues/1532) without prior notice, so we cannot support them as they are too fast of a moving target.
+- `21.3`+ (With `sentry-sdk` version 1.3.0 or higher)
```

Comment:
I'd add `21.3` to the list above without removing the general note. AFAIK there is no commitment to add support for every non-LTS release.

Reply:
The comment is highly misleading and not representative of the maintenance of the project. If there is a problem with the integration: ping me, I will help. 

https://sanicframework.org/en/guide/project/policies.html#deprecation

Reply:
> If the latest version of Sanic is not explicitly listed here, it _might_ not be supported.

Reply:
:+1: